### ISSUE TITLE
Select Cell + scroll not working

### ISSUE THREAD

**Issue by mmfilesi** [handsontable/handsontable#2700]
Hi : )

When the table have scroll, cell type not working

http://jsfiddle.net/csfkvt4v/

(click last item, colB).

**Comment by mmfilesi** [comment]
The first bug, editor always visible when scroll, can be fix with a hook

```javascript
        Handsontable.hooks.add('afterScrollVertically',function(){
            var editor = this.getActiveEditor();
            if ( editor && editor.hasOwnProperty('select') ) {
            	this.destroyEditor(false)
            }
        });
```

**Comment by budnix** [comment]
This bug should be fixed in 0.16.1. Can you confirm this in updated demo at http://jsfiddle.net/csfkvt4v/1/?

**Comment by mmfilesi** [comment]
PERFECT!

ty!!
